scripts/sanitycheck

```diff
@@ -1236,31 +1241,40 @@ class TestSuite:
             for dirpath, dirnames, filenames in os.walk(testcase_root,
                                                         topdown=True):
                 verbose("scanning %s" % dirpath)
-                if "sample.yaml" in filenames or "testcase.yaml" in filenames:
-                    verbose("Found possible test case in " + dirpath)
-                    dirnames[:] = []
-                    if "sample.yaml" in filenames:
-                        yaml_path = os.path.join(dirpath, "sample.yaml")
-                    else:
-                        yaml_path = os.path.join(dirpath, "testcase.yaml")
-                    cp = SanityConfigParser(yaml_path)
-                    workdir = os.path.relpath(dirpath, testcase_root)
+                if 'sample.yaml' in filenames:
+                    filename = 'sample.yaml'
+                elif 'testcase.yaml' in filenames:
+                    filename = 'testcase.yaml'
+                else:
+                    continue
+                verbose("Found possible test case in " + dirpath)
+                dirnames[:] = []
+                yaml_path = os.path.join(dirpath, filename)
+                try:
+                    cp = SanityConfigParser(yaml_path, scl.yaml_tc_schema)
+                except RuntimeError as e:
+                    error("E: %s: can't load: %s" % (yaml_path, e))
```

Comment:
conversation starter

The problem is error() is not printing any prefixes, so it is hard to tell what's a warning, what's an error, what's progress. I literally added that to be able to find the message while I was sprinkling errors to test.

I'd prefer error() and sibling prefix "Error:" "Warning:" or whatever (I prefer "[EWI]:" and then go from there, but I'd like your feedback on that first.